.github/workflows/check.yml

```diff
@@ -67,7 +67,7 @@ jobs:
 
       - name: Check Test
         run: |
-          cargo test
+          cargo test --no-fail-fast
```

Comment:
what does this to? reports error on first failure?

Reply:
`--no-fail-fast: Run all tests regardless of failure. Without this flag, Cargo will exit after the first executable fails.`
The CI at the end will fail anyway in case of any failed tests, but it will also run all the other tests too instead of stopping immediately. I guess you put this to avoid wasting time discovering new failing tests one after the other, right?

Reply:
Yup, it's so that all tests run so I avoid wasting time